app/models/document.rb

```diff
@@ -109,7 +109,7 @@ def self.from_publishing_api(payload)
       content_id: payload['content_id'],
       title: payload['title'],
       summary: payload['description'],
-      body: payload['details']['body'],
+      body: payload['details']['body'][0]['content'],
```

Comment:
I think we need to deal with the case where the body has the previous format of just a string, otherwise this will present a blank text field and overwrite the previous content.